Makefile

```diff
@@ -45,8 +66,40 @@ cockpit/download: Makefile
 
 .PHONY: cockpit/build
 cockpit/build: cockpit/download
-	npm ci
 	npm run build:cockpit
 
 .PHONY: cockpit/devel
 cockpit/devel: cockpit/devel-uninstall cockpit/build cockpit/devel-install
+
+#
+# Building packages
+#
+
+RPM_SPEC=cockpit/$(PACKAGE_NAME).spec
+RPM_TARBALL=rpmbuild/SOURCES/$(PACKAGE_NAME)-$(VERSION).tar.gz
```

Comment:
just `$(PACKAGE_NAME)-$(VERSION).tar.gz`, but package_name needs to be `cockpit-composer` now too.